src/ApiPort.VisualStudio/ApiPort.VisualStudio.csproj

```diff
@@ -31,7 +31,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <Prefer32Bit>false</Prefer32Bit>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
This was only enabled on Debug build and kept getting in the way of trying things out (such as unused field).